packages/core/src/components/DialogBox/DialogBox.test.tsx

```diff
@@ -31,6 +32,11 @@ describe('DialogBox component', () => {
         expect(container).toMatchSnapshot();
     });
 
+    it('should render properly with close icon when it is open', () => {
+        const { container } = dialogBoxRenderer({ open: true, minWidth: '200px', minHeight: '200px', showCloseIcon: true });
+        expect(container).toMatchSnapshot();
+    });
+
```

Comment:
Can you add a test that closes the Dialog box by firing a click event or simulating the close button press?